src/IRPrinter.cpp

```diff
@@ -627,7 +627,7 @@ void IRPrinter::visit(const Store *op) {
     do_indent();
     const bool has_pred = !is_one(op->predicate);
     if (has_pred) {
-        stream << "if (" << op->predicate << ") {\n";
+        stream << "predicate (" << op->predicate << ")\n";
```

Comment:
Sure we don't want braces here?

Reply:
OK, but I don't think there are other places in the .stmt language where we imply things by indentation alone. Not worth arguing over though.

(If we're gonna change the syntax of predicate-store here should we also make predicate-load similar?)